vector/src/main/res/layout/keybackup_setup_step2_fragment.xml

```diff
@@ -0,0 +1,151 @@
+        android:id="@+id/keybackupsetup"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        tools:context=".fragments.keybackupsetup.KeybackupSetupStep2Fragment">
+
+        <TextView
+            android:id="@+id/keybackupsetup_step2_text"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:layout_marginStart="20dp"
+            android:layout_marginTop="30dp"
+            android:layout_marginEnd="20dp"
+            android:text="@string/keybackup_setup_step2_text"
+            android:textAlignment="center"
+            app:layout_constraintBottom_toTopOf="@id/keybackup_passphrase_container"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toTopOf="parent" />
+
+
+        <android.support.constraint.ConstraintLayout
+            android:id="@+id/keybackup_passphrase_container"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:layout_marginTop="20dp"
+            android:layout_marginBottom="20dp"
+            app:layout_constraintBottom_toTopOf="@id/keybackupsetup_step2_button"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toBottomOf="@id/keybackupsetup_step2_text">
+
+            <LinearLayout
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:orientation="vertical">
+
+                <LinearLayout
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:layout_marginStart="16dp"
+                    android:layout_marginLeft="16dp"
+                    android:baselineAligned="false"
+                    android:orientation="horizontal">
+
+                    <android.support.design.widget.TextInputLayout
+                        android:id="@+id/keybackup_passphrase_enter_til"
+                        android:layout_width="0dp"
+                        android:layout_height="wrap_content"
+                        android:layout_weight="1"
+                        app:errorEnabled="true">
+
+                        <android.support.design.widget.TextInputEditText
+                            android:id="@+id/keybackup_passphrase_enter_edittext"
+                            android:layout_width="match_parent"
+                            android:layout_height="wrap_content"
+                            android:hint="@string/keybackup_setup_step2_passphrase_enter_hint"
+                            android:maxLines="3"
+                            android:textColor="?android:textColorPrimary" />
+
+                    </android.support.design.widget.TextInputLayout>
+
+                    <RelativeLayout
```

Comment:
This RelativeLayout should be removed

Reply:
it's the clickable area, what do you recommand?

Reply:
The image could be clickable